pg-commands/_template.md

```diff
@@ -0,0 +1,152 @@
+# ORDER BY
+
+<!---
+General structure:
+* Definition of keyword or function
+* Time-series focused friendly intro
+* Simple examples 
+* More complex examples 
+-->
+
+By default, in PostgreSQL and therefore TimescaleDB, the results from a basic
+`SELECT` query are unordered. You must include the [`ORDER BY`][orderby-postgres]
+clause to guarantee the order of your results. When you use `ORDER BY`, you need
+to specify the columns to order the results on then, optionally, give the
+ordering direction, either ascending or descending. If you don't specify the
+order direction for your columns, the default ordering is ascending.
+
+When you query time-series data, you need ordered results. Time occurs linearly
+and by nature is chronological. When you analyze and understand time-series
+data, you need to retain that order on time. To effectively analyze time-series
+data in TimescaleDB, the `ORDER BY` clause is necessary.
+
+## Syntax
+Single column ordering: sort all rows of data based on the values in the
+specified column:
+
+```sql
+ORDER BY <column_name> <optional ASC or DESC>
```

Comment:
this feels somewhat misleading and I'll note that we 
a) don't use this in a select clause it's just sort of floating out there, which maybe is good especially given the note below on other ways to use it, but also feels slightly weird and like we should acknowledge it somehow.
b) don't link to the SELECT clause documentation, when pretty much always this is going to be used with a SELECT
c) don't talk about the order of clauses anywhere in this document, which is often the biggest error and problem users have with using somethign like an order by clause with other clauses. 
d) don't mention that there are some other cases where order by clauses can be used outside of the standard ordering, two cases come up frequently: 1) window claues (ie PARTITION BY symbol ORDER BY time) and 2) in an aggregate expression, ie SELECT array_agg(symbol ORDER BY time) FROM stock_data will do something like this, and has the same syntax.